// setTimeout(()=>{
//     console.log("Hacking");
// }, 3000)

// try{
//     setTimeout(()=>{

//         console.log(rahul)
//     }, 100)
// }
// catch(err){
//     console.log("ther is some error " + err)
// }
// // console.log(rachul);
// setTimeout(()=>{
//     console.log("Hacking");
// }, 2000)
// setTimeout(()=>{
//     console.log("Hacking");
// }, 1000)
// setTimeout(()=>{
//     console.log("Hacking");
// }, 3000)
// setTimeout(()=>{
//     console.log("Hacking");
// }, 2000)
const f = () =>{

    try{
    //     let a = 0;
    //    console.log(program);
        console.log("sucessfully started");
        return
    }
    catch(error){
        console.log("error");
        console.log(p);
    }
    finally{
        console.log("finally started");
    }
}
f();